inabox/deploy/deploy.go

```diff
@@ -339,23 +339,23 @@ func (env *Config) StartBinaries() {
 // TODO: Supply the test path to the runner utility
 func (env *Config) StopBinaries() {
 	changeDirectory(filepath.Join(env.rootPath, "inabox"))
-	err := execCmd("./bin.sh", []string{"stop"}, []string{})
+	err := execCmd("./bin.sh", []string{"stop"}, []string{}, true)
 	if err != nil {
 		log.Panicf("Failed to stop binaries. Err: %s", err)
 	}
 }
 
 func (env *Config) StartAnvil() {
 	changeDirectory(filepath.Join(env.rootPath, "inabox"))
-	err := execCmd("./bin.sh", []string{"start-anvil"}, []string{})
+	err := execCmd("./bin.sh", []string{"start-anvil"}, []string{}, false) // printing output causes hang
```

Comment:
because too many prints?

Reply:
I am unsure what the root cause is. It may be something like that.